taichi/codegen/codegen_llvm.cpp

```diff
@@ -1479,61 +1474,44 @@ void CodeGenLLVM::visit(LinearizeStmt *stmt) {
 
 void CodeGenLLVM::visit(IntegerOffsetStmt *stmt){TI_NOT_IMPLEMENTED}
 
-llvm::Value *CodeGenLLVM::create_bit_ptr_struct(llvm::Value *byte_ptr_base,
-                                                llvm::Value *bit_offset) {
-  // 1. get the bit pointer LLVM struct
-  // struct bit_pointer {
-  //    i8* byte_ptr;
-  //    i32 offset;
+llvm::Value *CodeGenLLVM::create_bit_ptr(llvm::Value *byte_ptr,
+                                         llvm::Value *bit_offset) {
+  // 1. define the bit pointer struct (X=8/16/32/64)
+  // struct bit_pointer_X {
+  //    iX* byte_ptr;
+  //    i32 bit_offset;
   // };
+  TI_ASSERT(bit_offset->getType()->isIntegerTy(32));
   auto struct_type = llvm::StructType::get(
```

Comment:
OOC, where do we name `struct_type`?

Reply:
We don't name it - this is hashed by its element types in LLVM.